### PR TITLE
Add Gemini Embedding Model 001

### PR DESCRIPTION
**Pull request by NirantK**
Since there is limited nesting available on the left pane, when we create new folders inside Integration — even with `_index.md` they're no longer visible in the left pane. 

Solution: Split the Integrations into Embedding and Frameworks

image

### CODE REVIEW

**Comment by azayarni** [comment]
Please Make a new main Section "Integrations" parallel to "User Manual" and put there two sub sections: "Frameworks", "Embeddings".

**Comment by NirantK** [comment]
Did I understand you correctly @azayarni ? 

image

File: qdrant-landing/content/documentation/embedding/gemini.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+weight: 700
+---
+
+# Gemini
+
+Qdrant is compatible with Gemini Embedding Model API and its official Python SDK that might be installed as any other package:
+
+Gemini is a new family of Google PaLM models, released in December 2023. The new embedding models succeed the previous Gecko Embedding Model. 
+
+The new models allow passing an additional parameter to the API call: `task_type`. It determines the type of task you want to use the embeddings for.
```
Comment (atarora) [inline qdrant-landing/content/documentation/embedding/gemini.md:12]:
```suggestion
In the latest models, an additional parameter, task_type, can be passed to the API call. This parameter serves to designate the intended purpose for the embeddings utilized.

The Embedding Model API supports various task types, outlined as follows:
```

File: qdrant-landing/content/documentation/embedding/gemini.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+These are the task types that are supported by the Embedding Model API:
```
Comment (atarora) [inline qdrant-landing/content/documentation/embedding/gemini.md:14]:
```suggestion

```

File: qdrant-landing/content/documentation/embedding/gemini.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+- TASK_TYPE_UNSPECIFIED (0): Unset value, which will default to one of the other enum values.
+- RETRIEVAL_QUERY (1): Specifies the given text is a query in a search/retrieval setting.
+- RETRIEVAL_DOCUMENT (2): Specifies the given text is a document from the corpus being searched.
+- SEMANTIC_SIMILARITY (3): Specifies the given text will be used for STS.
+- CLASSIFICATION (4): Specifies that the given text will be classified.
+- CLUSTERING (5): Specifies that the embeddings will be used for clustering.
+
```
Comment (atarora) [inline qdrant-landing/content/documentation/embedding/gemini.md:22]: I suggest we lowercase the values as used with parameter "task_type" as used below.

File: qdrant-landing/content/documentation/embedding/gemini.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+Qdrant is compatible with Gemini Embedding Model API and its official Python SDK that might be installed as any other package:
+
+Gemini is a new family of Google PaLM models, released in December 2023. The new embedding models succeed the previous Gecko Embedding Model. 
+
+The new models allow passing an additional parameter to the API call: `task_type`. It determines the type of task you want to use the embeddings for.
+
+These are the task types that are supported by the Embedding Model API:
+
+- TASK_TYPE_UNSPECIFIED (0): Unset value, which will default to one of the other enum values.
```
Comment (atarora) [inline qdrant-landing/content/documentation/embedding/gemini.md:16]:
```suggestion
- task_type_unspecified: Unset value, which will default to one of the other values.
```

File: qdrant-landing/content/documentation/embedding/gemini.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+- RETRIEVAL_QUERY (1): Specifies the given text is a query in a search/retrieval setting.
```
Comment (atarora) [inline qdrant-landing/content/documentation/embedding/gemini.md:17]:
```suggestion
- retreival_query: Specifies the given text is a query in a search/retrieval setting.
```

File: qdrant-landing/content/documentation/embedding/gemini.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+- RETRIEVAL_DOCUMENT (2): Specifies the given text is a document from the corpus being searched.
```
Comment (atarora) [inline qdrant-landing/content/documentation/embedding/gemini.md:18]:
```suggestion
- retrieval_document: Specifies the given text is a document from the corpus being searched.
```

File: qdrant-landing/content/documentation/embedding/gemini.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+- SEMANTIC_SIMILARITY (3): Specifies the given text will be used for STS.
```
Comment (atarora) [inline qdrant-landing/content/documentation/embedding/gemini.md:19]:
```suggestion
- semantic_similarity: Specifies the given text will be used for Semantic Text Similarity.
```

File: qdrant-landing/content/documentation/embedding/gemini.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+- CLASSIFICATION (4): Specifies that the given text will be classified.
```
Comment (atarora) [inline qdrant-landing/content/documentation/embedding/gemini.md:20]:
```suggestion
- classification: Specifies that the given text will be classified.
```

File: qdrant-landing/content/documentation/embedding/gemini.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+- CLUSTERING (5): Specifies that the embeddings will be used for clustering.
```
Comment (atarora) [inline qdrant-landing/content/documentation/embedding/gemini.md:21]:
```suggestion
- clustering: Specifies that the embeddings will be used for clustering.
```

**Review by atarora — CHANGES_REQUESTED** [review]
Looks great ! Godo job @NirantK !!
Added some suggestions for review.

**Comment by azayarni** [comment]
Preview looks broken: https://deploy-preview-457--condescending-goldwasser-91acf0.netlify.app/documentation/embedding/

Default category page is empty, for both.

Category name "Embedding" => "Embeddings".

Also: txtAI, fiftyONe are in wrong category.

**Review by atarora — APPROVED** [review]

**Comment by NirantK** [comment]
Merging with @azayarni approval on Slack

**Pull request merged**